Sources/ViewInspector/Inspector.swift

```diff
@@ -310,9 +361,9 @@ internal extension Inspector {
         }
     }
     // swiftlint:enable cyclomatic_complexity
-    
+
     static func guardType(value: Any, namespacedPrefixes: [String], inspectionCall: String) throws {
-        
+
```

Comment:
@nalexn do you have a preference regarding the style of tabs / spaces / empty lines?
I can revert those to the original state if you prefer. Let me know.

Reply:
Nah, I like this more. [I've googled](https://stackoverflow.com/a/72840452/2923345) how to tweak the Xcode settings to not insert spaces

Reply:
@nalexn in case it's helpful, the [Airbnb style guide has a script](https://github.com/airbnb/swift#xcode-formatting) that you can run that enables various editor settings like this one that we find helpful in our large project.